test/csuite/tiered_abort/main.c

```diff
@@ -422,19 +440,23 @@ thread_run(void *arg)
                 locked = false;
             }
         }
+
+        /* We're done with the timestamps, allow oldest and stable to move forward. */
+        if (use_ts)
+            WT_PUBLISH(td->commit_ts, active_ts);
```

Comment:
Is this still required?

Reply:
This time around, yes. Now that we're writing to `td` after we're done with the timestamp, we don't want to allow any reordering between (say) setting the commit timestamp of the transaction, and storing this value in `td`.

Reply:
Got it.